runner/reporter.go

```diff
@@ -133,9 +134,17 @@ func newReporter(results chan *callResult, c *RunConfig) *Reporter {
 
 // Run runs the reporter
 func (r *Reporter) Run() {
+	var skipCount uint
+	if r.config.skipFirst > 0 {
+		println("Skipping the first", r.config.skipFirst, "requests from the results calculations.")
```

Comment:
Consider using `fmt.Println` here instead of `println`, as `println` is built in to the runtime and may be removed. See [this stackoverflow post](https://stackoverflow.com/questions/14680255/difference-between-fmt-println-and-println-in-go) for more detai.s